### PR TITLE
Support multiple keys via comma separated list

### PR DESCRIPTION
**Pull request by Evesy**
#### General

- [x] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass

#### Purpose

Adds support to pass multiple keys to `metrics-redis-llen.rb`, so that list length metrics can be gathered for multiple keys without the need to configure several checks.

All keys must belong to the same database.

#### Known Compatibility Issues

n/a

### CODE REVIEW

**Comment by Evesy** [comment]
Hopefully a basic snippet is sufficient for this change for now:

```
127.0.0.1:6379> llen alpha_key
(integer) 7
127.0.0.1:6379> llen beta_key
(integer) 3
```

```
$ /opt/sensu/embedded/bin/metrics-redis-llen.rb -k alpha_key
default-centos7.redis.alpha_key.items 7 1510265177
```

```
$ /opt/sensu/embedded/bin/metrics-redis-llen.rb -k alpha_key,beta_key
default-centos7.redis.alpha_key.items 7 1510265182
default-centos7.redis.beta_key.items 3 1510265182
```

**Comment by majormoses** [comment]
looks like we need to lock the version of `test-kitchen` as apparently newer versions required ruby 2.3. Looks like the last version that supports ruby < 2.3 is https://rubygems.org/gems/test-kitchen/versions/1.16.0 you can use `~> 1.16.0`

File: sensu-plugins-redis.gemspec [hunk 1]
```diff
@@ -47,7 +47,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop',                   '~> 0.40.0'
   s.add_development_dependency 'yard',                      '~> 0.8'
   s.add_development_dependency 'serverspec',                '~> 2.36.1'
-  s.add_development_dependency 'test-kitchen',              '~> 1.6'
+  s.add_development_dependency 'test-kitchen',              '~> 1.16.0'
```
Comment (majormoses) [inline sensu-plugins-redis.gemspec]: :+1:

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-redis/versions/2.3.0

**Pull request merged**